routing/portal_config.php

```diff
@@ -53,34 +42,24 @@ class Config
     public $phonedbName;
     public $phonedbUser;
     public $phonedbPass;
-    public function __construct($sitePath)
+    public function __construct($configData)
     {
-        $db = new PDO(
-            "mysql:host=".Routing_Config::$dbHost.";dbname=".Routing_Config::$dbName.";charset=UTF8",
-            Routing_Config::$dbUser,
-            Routing_Config::$dbPass,
-            array()
-        );
-        $sql = "SELECT a.*, b.database_name as phonedbName, b.path as orgchart_path_ext FROM portal_configs as a JOIN orgchart_configs as b ON a.orgchart_id = b.id WHERE a.path = '$sitePath';";
-        $query = $db->prepare($sql);
-        $query->execute(array());
-        $res = $query->fetchAll(PDO::FETCH_ASSOC);
-        $this->title = $res[0]['title'];
-        $this->city = $res[0]['city'];
-        $this->adminLogonName = $res[0]['adminLogonName'];    // Administrator's logon name
-        $this->adPath = json_decode($res[0]['active_directory_path']); // Active directory path
-        $this->uploadDir = ltrim($res[0]['path'], "/") . $res[0]['upload_directory'];
-        $this->fileManagerDir = ltrim($res[0]['path'], "/") . "files/"; // file manager directory
-        $this->portalPath = ltrim($res[0]['path'], '/');
+        $this->title = $configData[0]['title'];
+        $this->city = $configData[0]['city'];
+        $this->adminLogonName = $configData[0]['adminLogonName'];    // Administrator's logon name
+        $this->adPath = json_decode($configData[0]['active_directory_path']); // Active directory path
+        $this->uploadDir = ltrim($configData[0]['path'], "/") . $configData[0]['upload_directory'];
```

Comment:
Since we're storing the new UPLOADS location in leaf_config. Can we change this variable to be just ```ltrim($configData[0]['upload_directory'], '/')```

Reply:
done